src/test/test_releng_import_object.py

```diff
@@ -218,6 +218,10 @@ def test_genome(self):
             'ws_object_version/6:7:8',  # from
             'ws_type_version/KBaseGenomes.Genome-15.1'  # to
         )
+        # Check for type vertices
+        self.assertTrue(get_re_doc('ws_type_version', obj_type))
```

Comment:
I'd check the structure rather than they just exist

Reply:
Always calling out my laziness. Just updated to use assertDictContainsSubset